### ISSUE TITLE
No activity when prysm not run with special settings

### ISSUE THREAD

**Issue by stefa2k** [attestantio/vouch#6]
Errors:
```
{"level":"debug","service":"controller","impl":"standard","attestation_slot":13367,"current_slot":13369,"time":"2020-11-20T08:34:01Z","message":"Attestation for a past slot; not scheduling"}
```

```
{"level":"warn","strategy":"submitter","impl":"all","beacon_node_address":"eth2-test-node-b:5052","slot":13746,"error":"failed to submit beacon attestation: POST failed with status 400: {\"code\":400,\"message\":\"BAD_REQUEST: Invalid object: gossip verification failed: InvalidSignature\",\"stacktraces\":[]}","time":"2020-11-20T09:49:19Z","message":"Failed to submit attestation"}
```

```
{"level":"error","service":"attestationaggregator","impl":"standard","slot":13774,"attestation_data_root":"0xaef5e486c9c0bb8ba5a74b0899bcd1dba709c019d2cb5724fde04defae5c8082","error":"failed to submit aggregate and proofs: POST failed with status 400: {\"code\":400,\"message\":\"BAD_REQUEST: error processing aggregate and proofs\",\"failures\":[{\"index\":0,\"message\":\"Verification: InvalidSignature\"}]}","time":"2020-11-20T09:55:03Z","message":"Failed to submit aggregate and proof"}
```

Workaround:
Settings on prysm beacons:
```
--rpc-max-page-size=100000 --grpc-max-msg-size=268435456
```

**Comment by slayorktc** [comment]
Will this be fixed by genesis?

**Comment by mcdee** [comment]
Yes, it is currently under test with a few other changes that should be released in the next day or so.

**Comment by slayorktc** [comment]
I pulled in the latest changes but getting a different error now. Is this user error config or vouch bug?

{"level":"warn","service":"controller","impl":"standard","error":"failed to obtain attester duties: failed to request attester duties: GET failed with status 405: {\"code\":405,\"message\":\"METHOD_NOT_ALLOWED\",\"stacktraces\":[]}","time":"2020-11-27T18:43:19Z","message":"Failed to subscribe to beacon committees"}
{"level":"error","service":"controller","impl":"standard","error":"failed to request proposer duties: GET failed with status 503: {\"code\":503,\"message\":\"SERVICE_UNAVAILABLE: beacon node is syncing: head slot is 63104, current slot is 66848\",\"stacktraces\":[]}","time":"2020-11-27T18:49:43Z","message":"Failed to obtain proposer duties"}
{"level":"error","service":"controller","impl":"standard","error":"failed to request attester duties: GET failed with status 405: {\"code\":405,\"message\":\"METHOD_NOT_ALLOWED\",\"stacktraces\":[]}","time":"2020-11-27T18:49:43Z","message":"Failed to obtain attester duties"}
{"level":"warn","service":"controller","impl":"standard","error":"failed to obtain attester duties: failed to request attester duties: GET failed with status 405: {\"code\":405,\"message\":\"METHOD_NOT_ALLOWED\",\"stacktraces\":[]}","time":"2020-11-27T18:49:43Z","message":"Failed to subscribe to beacon committees"}

**Comment by mcdee** [comment]
The key line there is most likely:

```
 {"level":"error","service":"controller","impl":"standard","error":"failed to request proposer duties: GET failed with status 503: {"code":503,"message":"SERVICE_UNAVAILABLE: beacon node is syncing: head slot is 63104, current slot is 66848","stacktraces":[]}","time":"2020-11-27T18:49:43Z","message":"Failed to obtain proposer duties"}
```

I suspect that the other errors are due to the fact that the node is syncing.  Can you check again once the node is synced?

**Comment by slayorktc** [comment]
You are correct. All letting all nodes sync up, my issue is resolved.

**Comment by mcdee** [comment]
This issue was fixed with vouch 1.0.0
